packages/cli/src/commands/deploy.ts

```diff
@@ -111,6 +105,29 @@ export const deploy: DeployFunc = async ({
   )
 }
 
+const decidePackageName = async ({
+  name,
+  skipPrompts,
+  dir,
+}: {
+  name?: string
+  skipPrompts: boolean
+  dir: string
+}) => {
+  if (typeof name === 'string' && name.length > 0) {
+    return name
+  }
+
+  if (skipPrompts) {
+    return getPackageName(dir)
+  }
```

Comment:
Does this mean that 

- if I provide `--name` and `--skipPrompts`, this part is ignored, as line 118 will return.
- If I omitt `--name`, the package name will be used?

Reply:
- correct
- correct

because when you create a new package, you type the package name, and we put it inside package.json. So that's some default value we could rely on.